Tests/MessagingInApp/Util/ViewAnimationRunnerStub.swift

```diff
@@ -0,0 +1,19 @@
+@testable import CioMessagingInApp
+import Foundation
+import UIKit
+
+class ViewAnimationRunnerStub: ViewAnimationRunner {
```

Comment:
This is how we can mock view animations to make them synchronous which makes our tests more reliable and easier to write.